<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>QicsHTMLExportDialogClass</class>
 <widget class="QDialog" name="QicsHTMLExportDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>QicsHTMLExportDialog</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="3" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Target</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>HTML file name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leFileName" />
      </item>
      <item>
       <widget class="QToolButton" name="tbPickPath" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>181</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1" >
    <widget class="QPushButton" name="bOk" >
     <property name="text" >
      <string>OK</string>
     </property>
     <property name="default" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2" >
    <widget class="QPushButton" name="bCancel" >
     <property name="text" >
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3" >
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Images</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="4" column="0" >
       <widget class="QComboBox" name="cbPixFormat" >
        <property name="editable" >
         <bool>true</bool>
        </property>
        <property name="insertPolicy" >
         <enum>QComboBox::InsertAfterCurrent</enum>
        </property>
        <item>
         <property name="text" >
          <string>PNG</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>GIF</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>JPG</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>JPEG</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>BMP</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>PPM</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>XPM</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>XBM</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QSpinBox" name="sbPixQuality" >
        <property name="specialValueText" >
         <string>Default</string>
        </property>
        <property name="maximum" >
         <number>100</number>
        </property>
        <property name="minimum" >
         <number>-1</number>
        </property>
        <property name="value" >
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Size/quaility (0-smallest, 100-finest):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="leWidgetName" >
        <property name="text" >
         <string>wid-%1-%2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLineEdit" name="lePixName" >
        <property name="text" >
         <string>pix-%1-%2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QCheckBox" name="cbWidgets" >
        <property name="text" >
         <string>Export widgets (as images)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Image format:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Image naming template (%1 - cell row, %2 - cell column):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="cbImages" >
        <property name="text" >
         <string>Export images</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>HTML Options</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="cbHeaders" >
        <property name="text" >
         <string>Export headers</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QCheckBox" name="cbColors" >
        <property name="text" >
         <string>Export cell/text coloring</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QCheckBox" name="cbFlags" >
        <property name="text" >
         <string>Export text flags</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QCheckBox" name="cbAlignment" >
        <property name="text" >
         <string>Export cell alignment</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QCheckBox" name="cbSelection" >
        <property name="text" >
         <string>Highlight active selection</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QCheckBox" name="cbFillEmpty" >
        <property name="text" >
         <string>Fill empty cells with &amp;&amp;nbsp</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="cbDimensions" >
        <property name="text" >
         <string>Export cell dimensions</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QCheckBox" name="cbFont" >
        <property name="text" >
         <string>Export font data</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QCheckBox" name="cbCellBorders" >
        <property name="text" >
         <string>Export cell borders</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QCheckBox" name="cbTableBorders" >
        <property name="text" >
         <string>Export table borders</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QGroupBox" name="groupBox_4" >
     <property name="title" >
      <string>Source</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rbTable" >
        <property name="text" >
         <string>Whole table</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbSelected" >
        <property name="text" >
         <string>Selected area</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbData" >
        <property name="text" >
         <string>Data area</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>leFileName</tabstop>
  <tabstop>tbPickPath</tabstop>
  <tabstop>cbImages</tabstop>
  <tabstop>cbWidgets</tabstop>
  <tabstop>lePixName</tabstop>
  <tabstop>leWidgetName</tabstop>
  <tabstop>cbPixFormat</tabstop>
  <tabstop>sbPixQuality</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cbWidgets</sender>
   <signal>toggled(bool)</signal>
   <receiver>leWidgetName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>309</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel" >
     <x>309</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbImages</sender>
   <signal>toggled(bool)</signal>
   <receiver>lePixName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>66</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel" >
     <x>102</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bOk</sender>
   <signal>clicked()</signal>
   <receiver>QicsHTMLExportDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>254</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel" >
     <x>140</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bCancel</sender>
   <signal>clicked()</signal>
   <receiver>QicsHTMLExportDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>332</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel" >
     <x>35</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
